tests/foreman/ui/test_usergroup.py

```diff
@@ -50,29 +47,15 @@ def test_negative_create_usergroup_with_too_long_name(self, group_name):
 
         """
         with Session(self.browser) as session:
-            make_usergroup(
-                session, org=self.organization.name, name=group_name)
-            self.assertIsNotNone(self.usergroup.wait_until_element(
-                common_locators['name_haserror']))
-            self.assertIsNone(self.usergroup.search(group_name))
-
-    @data('', ' ')
-    def test_negative_create_usergroup_with_blank_name(self, group_name):
```

Comment:
I combined this test and the previous one to create a new invalid test which uses invalid_values_list() helper.